DailyScreenshot/DailyScreenshot.csproj

```diff
@@ -57,6 +58,7 @@
     <None Include="manifest.json" />
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup />
```

Comment:
Think this may have been added automatically or by accident, isn't needed here. Going to do same thing as with the typo; will fix later since it's minor and not worth delaying the PR getting merged.